column/api/controller/run_controller.py

```diff
@@ -15,20 +15,27 @@
 LOG = logging.getLogger(__name__)
 
 run_post_schema = {
-    'playbook_path': {'type': 'string', 'required': True},
-    'inventory_file': {'type': 'string'},
-    'options': {
-        'type': 'dict',
-        'schema': {
-            'become_user': {'type': 'string'},
-            'private_key_file': {'type': 'string'},
-            'tags': {'type': 'list'},
-            'skip_tags': {'type': 'list'},
-            'susbset': {'type': 'string'},
-            'connection': {'type': 'string'},
-            'become_method': {'type': 'string'}
+    "$schema": "http://json-schema.org/schema#",
+    "type": "object",
+    "properties": {
+        "playbook_path": {"type": "string"},
+        "inventory_file": {"type": "string"},
+        "options": {
+            "type": "object",
+            "properties": {
+                "become_user": {"type": "string"},
+                "private_key_file": {"type": "string"},
```

Comment:
Not sure we should be allowing the path name to a file as part of the API. Should be private key itself.